LICENCE.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (C) 2013 Crown Copyright (Government Digital Service)
```

Comment:
I'd normally include a list or range of years, but just because I've seen this used elsewhere.